src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,152 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
+    public MyLinkedList() {
+        this.head = null;
+        this.tail = null;
+        this.size = 0;
```

Comment:
these are default values, we don't need to set them explicitly, co you can remove constructor

Reply:
done, removed the constructor and made this values default